lib/datadog/tracing/contrib/action_pack/action_dispatch/instrumentation.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require_relative '../../../metadata/ext'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module ActionPack
+        module ActionDispatch
+          # Instrumentation for ActionDispatch components
+          module Instrumentation
+            module_function
+
+            def set_http_route_tag(http_route)
+              return if http_route.nil?
+
+              active_span = Tracing.active_span
+              return unless active_span
+
+              active_span.set_tag(
+                Tracing::Metadata::Ext::HTTP::TAG_ROUTE,
+                http_route.gsub(/\(.:format\)$/, '')
+              )
+            end
+
+            # Instrumentation for ActionDispatch::Journey components
+            module Journey
+              # Instrumentation for ActionDispatch::Journey::Router
+              # for Rails versions older than 7.1
+              module Router
+                def find_routes(req)
+                  result = super(req)
+
+                  return response unless Tracing.enabled?
```

Comment:
`return result` ?

Reply:
And it seems like there perhaps isn't test coverage for this path.

Reply:
thank you for finding this! Yes, there is none, I am working to add appraisals for different action pack versions. I am a bit afraid to merge such changes without coverage (especially after this find).

Reply:
Added appraisals for different action_pack versions here [59b5290](https://github.com/DataDog/dd-trace-rb/pull/3849/commits/59b52908dfe533ff6d93e80270b7eba709e4413f)

Reply:
Appraisals are fixed in this comment: https://github.com/DataDog/dd-trace-rb/pull/3849/commits/5ba0feff11e20c684148c18eb103aaa41d79b2dc